public/privacy.html

```diff
@@ -0,0 +1,82 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+   <link rel='shortcut icon' type='image/x-icon' href='/img/favicon.ico' />
+    <meta charset="UTF-8">
+    <title>MissingKids Slack Bot - Privacy Policy</title>
+	</head>
+	<script src="/scripts/jquery-1.7.1.min.js"></script>
+    <script src="/scripts/url.min.js"></script>
+    <script src="/scripts/register.js"></script>
+    <script src="/scripts/stats.js"></script>
```

Comment:
It's my bad, remove it from both please. Watch the console for any other errors.